modules/standard/BigInteger.chpl

```diff
@@ -3485,120 +3527,168 @@ module BigInteger {
+  proc addMul(ref result: bigint, const ref x: bigint, y: int) {
+    if y >= 0
+      then addMul(result, x, y:uint);
+      else subMul(result, x, (0 - y):uint);
   }
 
-  proc addmul(ref result: bigint, const ref a: bigint, b: int) {
-    if b >= 0 {
-      BigInteger.addmul(result, a, b:uint);
-    } else {
-      BigInteger.submul(result, a, (0 - b):uint);
-    }
-  }
-
-  @deprecated(notes="bigint.addmul method is deprecated - please use the standalone function :proc:`~BigInteger.addmul`")
-  proc bigint.addmul(const ref a: bigint, b: int) {
-    BigInteger.addmul(this, a, b);
-  }
-
-  proc addmul(ref result: bigint, const ref a: bigint, b: uint) {
-    const b_ = b.safeCast(c_ulong);
+  /* See :proc:`addMul` */
+  proc addMul(ref result: bigint, const ref x: bigint, y: uint) {
+    const y_ = y.safeCast(c_ulong);
 
     if compiledForSingleLocale() {
-      mpz_addmul_ui(result.mpz, a.mpz, b_);
+      mpz_addmul_ui(result.mpz, x.mpz, y_);
     } else if result.localeId == chpl_nodeID {
-      const a_ = a;
-      mpz_addmul_ui(result.mpz, a_.mpz, b_);
+      const x_ = x;
+      mpz_addmul_ui(result.mpz, x_.mpz, y_);
     } else {
       const resultLoc = chpl_buildLocaleID(result.localeId, c_sublocid_any);
       on __primitive("chpl_on_locale_num", resultLoc) {
-        const a_ = a;
-        mpz_addmul_ui(result.mpz, a_.mpz, b_);
+        const x_ = x;
+        mpz_addmul_ui(result.mpz, x_.mpz, y_);
       }
     }
   }
 
-  @deprecated(notes="bigint.addmul method is deprecated - please use the standalone function :proc:`~BigInteger.addmul`")
-  proc bigint.addmul(const ref a: bigint, b: uint) {
-    BigInteger.addmul(this, a, b);
-  }
+  @deprecated(notes="bigint.submul method is deprecated - please use the standalone function :proc:`subMul` instead")
+  proc bigint.submul(const ref a: bigint, const ref b: bigint)
+    do subMul(this, a, b);
+
+  @deprecated(notes="bigint.submul method is deprecated - please use the standalone function :proc:`subMul` instead")
+  proc bigint.submul(const ref a: bigint, b: int)
+    do subMul(this, a, b);
+
+  @deprecated(notes="bigint.submul method is deprecated - please use the standalone function :proc:`subMul` instead")
+  proc bigint.submul(const ref a: bigint, b: uint)
+    do subMul(this, a, b);
 
-  proc submul(ref result: bigint, const ref a: bigint, const ref b: bigint) {
+  @deprecated(notes="submul is deprecated - please use :proc:`subMul` instead")
+  proc submul(ref result: bigint, const ref a: bigint, const ref b: bigint)
+    do subMul(result, a, b);
+
+  @deprecated(notes="submul is deprecated - please use :proc:`subMul` instead")
+  proc submul(ref result: bigint, const ref a: bigint, b: int)
+    do subMul(result, a, b);
+
+  @deprecated(notes="submul is deprecated - please use :proc:`subMul` instead")
+  proc submul(ref result: bigint, const ref a: bigint, b: uint)
+    do subMul(result, a, b);
+
+  /* Subtracts the product of ``x`` and ``y`` from ``result``
```

Comment:
This (and one other case above) we have a space after the `/*`, which is better than it missing, but I think we should try to keep all those comment blocks consistent, where my preference would be for the newline after the `/*`.